app/renderer/ui/preference-view/components/extension-setting.vue

```diff
@@ -15,9 +16,19 @@ const props = defineProps({
     },
     required: true,
   },
+  name: { type: String, required: true },
 });
 
 const emits = defineEmits(["event:change"]);
+
+const onButtonClick = () => {
+  PLAPI.logService.info(
+    props.name,
```

Comment:
why do we need this `name`?

and the log should be removed?

Reply:
- Users need feedback when the button is clicked, which is why the log should be kept. 
- The `name` tells where the log is from.

![image](https://github.com/user-attachments/assets/ccd78acc-d339-4e40-8ceb-fcdf75633cd5)

Reply:
I think we don't need to trigger the log here.

We can do it in the extension code.

The extension developer listens to this button in the extension code, once user click it, run corresponding function, such as clearing the cache, and how a log that the cache has been cleared.

Reply:
In [7211188](https://github.com/Future-Scholars/paperlib/pull/606/commits/7211188ac41673c5f208e8ff4c09637bce259950) and [bbe480e](https://github.com/Future-Scholars/paperlib-ai-chat-extension/commit/bbe480ee360ddd7f15e6d2ce2eb04ac189a26e3a), the extension will listen to the change of preference value. Now the log can be removed.